package.json

```
@@ -25,10 +25,6 @@
   },
   "homepage": "https://github.com/remcohaszing/gatsby-remark-mermaid#readme",
   "dependencies": {
-    "mermaid": "^8.7.0",
-    "unist-util-visit": "^1.4.0"
-  },
-  "peerDependencies": {
-    "puppeteer": "*"
+    "remark-mermaidjs": "^3.1.0"
```

I just created a new release of `remark-mermaidjs`. Let’s use that here. It works pretty much the same ([release notes](https://github.com/remcohaszing/remark-mermaidjs/releases/tag/v4.0.0))

```suggestion
    "remark-mermaidjs": "^4.0.0"
```